packages/docs-site/src/content/docs/docs/integrations/overview.mdx

```diff
@@ -21,6 +21,14 @@ Legend:
 - 🗓️ Planned 
 - ❌ Not Supported
 
+### [Remix](/docs/integrations/remix/)
```

Comment:
support is looking pretty good across the board, so I think we may want to re-think this page